sample-code/spring-app/src/test/java/com/sap/ai/sdk/app/controllers/OrchestrationTest.java

```diff
@@ -214,10 +216,21 @@ void testInputFilteringStrict() {
     var policy = AzureFilterThreshold.ALLOW_SAFE;
 
     assertThatThrownBy(() -> service.inputFiltering(policy))
-        .isInstanceOf(OrchestrationClientException.class)
         .hasMessageContaining(
             "Prompt filtered due to safety violations. Please modify the prompt and try again.")
-        .hasMessageContaining("400 Bad Request");
+        .hasMessageContaining("400 (Bad Request)")
+        .isInstanceOfSatisfying(
+            OrchestrationInputFilterException.class,
+            e -> {
+              assertThat(e.getFilterDetails()).isNotNull();
+              assertThat(e.getFilterDetails()).containsKey("azure_content_safety");
+              assertThat(e.getFilterDetails().get("azure_content_safety")).isInstanceOf(Map.class);
+
+              var actualAzureContentSafety =
+                  (Map<String, Integer>) e.getFilterDetails().get("azure_content_safety");
```

Comment:
Please update the test to `use getAzureFilterInput()`, same for the output and LlamaGuard.